app/src/main/java/com/hfad/beeradviser/FindBeerActivity.java

```diff
@@ -28,9 +29,10 @@ public void onClickFindBeer(View view) {
         String beerType = String.valueOf(colorSpinnerView.getSelectedItem());
 
         //TODO: WRITE code to get recommendations from the BeerExpert class
-
+        expert.setColor(beerType);
+        String beerRecomendations = expert.getRecommendation(beerType);
```

Comment:
Since the expert object already knows the beerType, refactor getRecommendation() to require no arguments